python/ray/air/util/node.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional, Union
+import ray
+
+
+def _get_node_id_from_node_ip(node_ip: str) -> Optional[str]:
+    """Returns the node ID for the first alive node with the input IP."""
+    for node in ray.nodes():
+        if node["Alive"] and node["NodeManagerAddress"] == node_ip:
+            return node["NodeID"]
+
+    return None
+
+
+def _force_on_node(
+    node_id: str,
+    remote_func_or_actor_class: Optional[
+        Union[ray.remote_function.RemoteFunction, ray.actor.ActorClass]
+    ] = None,
+) -> Optional[Union[ray.remote_function.RemoteFunction, ray.actor.ActorClass]]:
```

Comment:
Maybe we can also add a type alias for the 2nd one?

```suggestion
) -> Union[Dict, Optional[Union[ray.remote_function.RemoteFunction, ray.actor.ActorClass]]]:
```

Reply:
Oof, good catch.